decosjoin/api/decosjoin/decosjoin_connection.py

```diff
@@ -273,6 +274,9 @@ def _transform(self, zaken, user_identifier):  # noqa: C901
                 ]
                 new_zaak['decision'] = _get_translation(new_zaak['decision'], translations)
 
+                if not bool(new_zaak['title']):
```

Comment:
Is dit safe? Kun je niet beter `new_zaak['title'] is not None and len(new_zaak['title'].strip())>1` @drakonen Wat zou jij doen?

Reply:
Title is gegarandeerd een string of None. (want to_string doet dat) (( er is ook `to_string_or_empty_string` als je echt een string wilt ))
`to_string` doet al een `strip()`
Een cast naar bool is niet nodig, een lege string in python is falsy.

maar moet de naam niet altijd `Gehandicapten parkeerplek` zijn?
dan hoeft het niet in de fields en kan het gewoon erna assigned worden zonder if statement.

Reply:
Alright, dus je kunt ook `if (not new_zaak['title']) { ... }` doen?

Reply:
Het moet niet altijd `Gehandicapten parkeerplek ` zijn dit is meer een fallback wanneer er geen title gezet is(maar als we deze logica hier doen dan denk ik dat we dit bij alle velden moeten doen) Ik wil eerst nog meer info van Micha wat de verwachtingen zijn van dit veld.